tcf_core/settings/base.py

```diff
@@ -166,6 +166,7 @@
     'social_core.pipeline.social_auth.social_details',
     'social_core.pipeline.social_auth.social_uid',
     'tcf_core.auth_pipeline.auth_allowed',
+    'tcf_core.auth_pipeline.validate_email',
```

Comment:
![image](https://github.com/thecourseforum/theCourseForum2/assets/53921230/1149ac47-887e-4ede-a85c-ba61058c504d)


I think the issue is that `tcf_core.auth_pipeline.validate_email` isn't meant to be used in `SOCIAL_AUTH_PIPELINE`. According to [docs](https://python-social-auth.readthedocs.io/en/latest/pipeline.html#email-validation), to set up email validation you:
1) Add `social_core.pipeline.mail.mail_validation` to `SOCIAL_AUTH_PIPELINE` (already being done [on this line](https://github.com/thecourseforum/theCourseForum2/blob/52cb9f74a8c9649b44e4dd0bb2bd752b3618dae1/tcf_core/settings/base.py#L172))

2) Add a custom function to `SOCIAL_AUTH_EMAIL_VALIDATION_FUNCTION`. In the existing code, `tcf_core.auth_pipeline.validate_email` is used for this [on this line](https://github.com/thecourseforum/theCourseForum2/blob/52cb9f74a8c9649b44e4dd0bb2bd752b3618dae1/tcf_core/settings/base.py#L157)